src/Sepes.Infrastructure/Service/Queries/StudyBaseQueries.cs

```diff
@@ -23,18 +22,59 @@ public static IQueryable<Study> StudyDetailsQueryable(SepesDbContext db)
         {
             return ActiveStudiesBaseQueryable(db)
                 .Include(s => s.StudyParticipants)
-                .ThenInclude(s => s.User)
+                .ThenInclude(sp=> sp.User)
                  .Include(s => s.Sandboxes)
                 .Include(s => s.StudyDatasets)
                     .ThenInclude(sd=> sd.Dataset)
                         .ThenInclude(sd => sd.SandboxDatasets)
-                            .ThenInclude(sd => sd.Sandbox);
+                            .ThenInclude(sd => sd.Sandbox)
+                            .AsNoTracking();
         }
 
-        public static IQueryable<Study> AllStudiesWithParticipantsQueryable(SepesDbContext db)
+        public static IQueryable<Study> StudyParticipantOperationsQueryable(SepesDbContext db)
         {
-            return db.Studies
-                .Include(s => s.StudyParticipants);
+            return ActiveStudiesBaseQueryable(db)
+                .Include(s => s.StudyParticipants)
+                .Include(s => s.Sandboxes)
+                     .ThenInclude(sd => sd.Resources)
+                .Include(s => s.Resources);
+             
+        }
+
+        public static IQueryable<Study> StudyDatasetsQueryable(SepesDbContext db)
+        {
+            return ActiveStudiesBaseQueryable(db)
+                .Include(s => s.StudyParticipants)
+                      .ThenInclude(sp => sp.User)
+                .Include(s => s.StudyDatasets)
+                    .ThenInclude(sd => sd.Dataset)
+                     .ThenInclude(sd => sd.Resources);
+                      
+        }
+
+        public static IQueryable<Study> StudySandboxCreationQueryable(SepesDbContext db)
+        {
+            return ActiveStudiesBaseQueryable(db)
+                .Include(s => s.StudyParticipants)
+                    .ThenInclude(sp=> sp.User)
+                .Include(s => s.Sandboxes);
+        }
+
+        public static IQueryable<Study> StudyDeleteQueryable(SepesDbContext db)
+        {
+            return ActiveStudiesWithParticipantsQueryable(db)
+                .Include(s => s.Sandboxes)
+                .Include(s=> s.Resources)
+                    .ThenInclude(r=> r.ChildResources)
+                .Include(s => s.StudyDatasets)
+                    .ThenInclude(sds=> sds.Dataset);
+        }
+
+        public static IQueryable<Study> StudyDatasetCreationQueryable(SepesDbContext db)
+        {
+            return StudyDatasetsQueryable(db)
+                .Include(s => s.Resources)
+                .ThenInclude(r=> r.Operations);
         }
```

Comment:
Would GraphQL be another way of doing this? If you have heard about it

Reply:
Not exactly, that would be to add more flexibility to external customers of the api? But these queries are for internal usage by the program